DQMOffline/EGamma/plugins/PhotonDataCertification.cc

```diff
@@ -85,17 +85,18 @@ float PhotonDataCertification::invMassZtest(string path, TString name,
   if (TestElem == 0) return 0;
   TH1F *TestHist = TestElem->getTH1F();
   if (TestHist == 0) return 0;
+  RooMsgService::instance().setGlobalKillBelow(RooFit::WARNING) ;
```

Comment:
Calling `RooMsgService::instance()` is not thread safe
https://root.cern.ch/doc/master/RooMsgService_8cxx_source.html#l00361